Common/Python/PandasConverter.cs

```diff
@@ -189,7 +193,7 @@ public PandasData(IBaseData baseData)
         /// <param name="baseData"><see cref="IBaseData"/> object that contains information to be saved in an instance of <see cref="PandasData"/></param>
         public void Add(IBaseData baseData)
         {
-            _timeIndex.Add(baseData.Time);
+            _timeIndex.Add(baseData.EndTime);
```

Comment:
I was hoping for a little discussion around this change, @jaredbroad -- any comments?

One issue I see is with daily data. To access data *for* to 30th of October I would need to enter a date of the 31st

Reply:
I think we should go for EndTime. If the users makes a History request to warm up and indicator, he should use EndTime in Indicator.Update method. If we give him Time. he would need to discount the Period.

Reply:
That's a pretty good point, also, I think it's rare people are accessing the data frame by date/time